plugins/woocommerce/client/legacy/js/admin/woocommerce_admin.js

```diff
@@ -361,6 +361,26 @@
 			$( this ).trigger( 'focus' );
 		} );
 
+		$( '#set-post-thumbnail' )
+			.parent()
+			.append(
+				'<span class="woocommerce-help-tip" tabindex="-1"></span>'
+			)
+			.find( '.woocommerce-help-tip' )
```

Comment:
I'm not a jQuery expert (I looked this up), but I think you could do...

```
$( '#set-post-thumbnail` ).after( '<span class="woocommerce-help-tip" tabindex="-1"></span>' )
```

That would be a little more clearer, I think.